src/lib/command-helpers.mjs

```diff
@@ -54,8 +54,10 @@ const COMMON_QUERY_OPTIONS = {
   },
   accountKey: {
     type: "string",
-    description: "The account key to use when calling Fauna",
+    description:
+      "The key to use for calling the Fauna Account API. Providing an account key will negate the need for a user login. The key will be used to generate short-lived database secrets. Cannot be used with --user or --secret.",
```

Comment:
IMO we don't need to call out the Fauna Account API here. Some other wording nits to align with other help.
```suggestion
      "Fauna account key used for authentication. Negates the need for a user login. The key is used to generate short-lived database secrets for the CLI. Mutually exclusive with `--user` and `--secret`.",
```